src/api/constants.ts

```diff
@@ -1,23 +1,25 @@
 import { Configuration, ProtocolMode } from '@azure/msal-browser';
+import {
+  REACT_APP_CLIENT_ID,
+  REACT_APP_AUTHORITY,
+  BASE_APP_URL,
+} from '../windows';
 
-export const CLIENT_ID = '7RswiAboRELefuFhVK9RRhaWloTr0VNq';
-export const AUDIENCE = 'https://dev-y4thf51u18ja0ber.eu.auth0.com/api/v2/';
 export const SCOPE = ['openid', 'offline_access', 'email'];
 export const HEADERS = { 'content-type': 'application/json' };
-export const BASE_URL = 'https://dev-y4thf51u18ja0ber.eu.auth0.com';
 
 export const msalConfig: Configuration = {
   auth: {
-    clientId: CLIENT_ID,
-    authority: BASE_URL,
-    redirectUri: 'http://localhost:3000/home',
-    postLogoutRedirectUri: 'http://localhost:3000/',
+    clientId: REACT_APP_CLIENT_ID,
+    authority: REACT_APP_AUTHORITY,
+    redirectUri: `${BASE_APP_URL}home`,
```

Comment:
Usually, it's better to define URLs as a complete address without the slash '/' at the end, so that when you need to construct a complete url, you would know that it should start with a slash. For example, I would do it like so:
string baseUrl = http://localhost:3000
string resource1 = `{baseUrl}/resource1` - the path to resourse 1 is '/resource1'
string resource2 = `{baseUrl}/resource2` - the path to resource 2 is '/resource2'

You would want to have your relative paths always starting with '/path-to-resource'. It think that this is the better approach.
Have in mind that https://localhost:3000 means the baseUrl, but when you attach '/' at the end, this means the 'root' path of the baseUrl. For this reason you would want to set your baseUrl without a slash.